itou/cities/management/commands/sync_cities.py

```diff
@@ -61,6 +91,77 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument("--wet-run", dest="wet_run", action="store_true")
 
+    def handle_deletions(self, cities_removed_by_api):
+        relations_to_fix = [
+            (JobDescription, "location"),
+            *((model, "insee_city") for model in AddressMixin.__subclasses__()),
+        ]
+        model_to_refill_infos = {}
+        for model, field_name in relations_to_fix:
+            refill_infos = {
+                pk: (previous_insee_code, last_synced_at)
+                for pk, previous_insee_code, last_synced_at in model.objects.filter(
+                    **{f"{field_name}__code_insee__in": cities_removed_by_api}
+                ).values_list("pk", f"{field_name}__code_insee", f"{field_name}__last_synced_at")
+            }
+            model.objects.filter(pk__in=refill_infos).update(**{field_name: None})
+            self.logger.info(f"Removed {field_name} from=%d {model.__name__} due to city deletion", len(refill_infos))
```

Comment:
Pour éviter les incompréhensions, en première lecture des logs j'étais pas sûr de si c'est une PK ou pas :sweat_smile: et ça permet de rester cohérent avec les autres logs :
```suggestion
            self.logger.info(f"Removed {field_name} from count=%d {model.__name__} due to city deletion", len(refill_infos))
```